tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,27 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: as a sharee search a file tagged by sharee using tag
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has uploaded file with content "hello world" to "uploadFolder/file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "uploadFolder/file2.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "uploadFolder/file3.txt"
+    And user "Alice" has shared folder "/uploadFolder" with user "Brian"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    And user "Brian" has created the following tags for file "uploadFolder/file1.txt" of the space "Shares":
+      | tag1 |
+    And user "Brian" has created the following tags for file "uploadFolder/file2.txt" of the space "Shares":
+      | tag1 |
+    When user "Brian" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Brian" should contain only these files:
```

Comment:
Should user `Alice` also search using `Tags:tag1`

Reply:
The would be nice if it gives the response.

Reply:
@PrajwolAmatya IMO not needed here, Scenario 11 will cover that.